requirements/testing.txt

```diff
@@ -84,7 +84,7 @@ pyfakefs==4.5.6
     # via -r requirements/testing.in
 pyhive[presto]==0.6.5
     # via apache-superset
-pylint==2.17.3
+pylint==2.9.6
```

Comment:
Seems like using a newer version of pylint runs up against a bunch of problems linting with the base superset repo.

Reply:
You must control these dependencies in testing.in file.

Reply:
Not sure if this is followed by the superset guys: 


They manually bump the pylint dependency here:
https://github.com/apache/superset/commit/420dd5b94a6c38192208c39bad899aeaa6bb6dbb


and straight up got rid of the dependency two days ago:

https://github.com/apache/superset/commit/9fd936c18c854a5a3c2f14c5cf13bd49e0fda6ab


As of today, the github actions runner is still using 2.9.6

https://github.com/apache/superset/actions/runs/4941268247/jobs/8833730890


I'm genuinely confused if there's a structure to this or not.

Reply:
I've been using pip-compile-multi to regenerate the dependencies, so if we don't properly set the dependency in testing.in then the changes in testing.txt might be overwritten next time I update the packages.

Reply:
Sure, I hardcoded in the versions that are pointed to in the main branch repo,

`pylint==2.9.6` 
`wrapt==1.12.0`

into the .in files for now until I can get a better understanding of what they're doing over there.